pallets/attesters/src/lib.rs

```diff
@@ -151,6 +203,24 @@ pub mod pallet {
     pub type PendingSlashes<T: Config> =
         StorageMap<_, Identity, T::AccountId, Vec<Slash<T::BlockNumber>>>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn attestation_targets)]
+    pub type AttestationTargets<T: Config> = StorageValue<_, Vec<TargetId>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn next_batches)]
+    pub type NextBatch<T: Config> = StorageMap<_, Identity, TargetId, BatchMessage<T::BlockNumber>>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn batches_to_sign)]
+    pub type BatchesToSign<T: Config> =
+        StorageMap<_, Identity, TargetId, Vec<BatchMessage<T::BlockNumber>>>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn batches)]
+    pub type Batches<T: Config> =
+        StorageMap<_, Identity, TargetId, Vec<BatchMessage<T::BlockNumber>>>;
```

Comment:
These storage items are unbounded by the value, should we try to fix the size to some theoretical limit here?

Reply:
will maybe extract it to separate issue as it possibly applies to all of StorageMaps across the project?